README.md

```diff
@@ -1,9 +1,27 @@
 # activate-power-mode atom package
 A work in progress package for Atom to replicate [codeinthedark/editor#1](https://github.com/codeinthedark/editor/pull/1) by popular demand.
 
-Activate with <kbd>Ctrl</kbd>-<kbd>Alt</kbd>-<kbd>O</kbd> or through the command panel with `Activate Power Mode: Toggle`. In
-this early version it only affects the current tab, to run it in a new tab you need to run `Window: Reload` first.
-
 **NOTE THAT THIS VERSION IS VERY BUGGY RIGHT NOW**
 
 ![activate-power-mode](https://cloud.githubusercontent.com/assets/688415/11453297/b8f249ec-9605-11e5-978c-eb3bb21eecd8.gif)
+
+## Installation
+See the [contributing](#contributing) section. This package will be available through apm once it's stable.
+
+## Usage
+Activate with <kbd>Ctrl</kbd>-<kbd>Alt</kbd>-<kbd>O</kbd> or through the command panel with `Activate Power Mode: Toggle`. In
+this early version it only affects the current tab. To run it in a new tab you need to run `Window: Reload` first.
+
+## Contributing
+If you haven't already done so, install [Node.js](https://nodejs.org/).
+
+Once Node.js is installed, place this package in your `~/.atom/packages/` directory. It's recommended that you use git to clone this package into that directory, as follows:
+
+    git clone https://github.com/JoelBesada/activate-power-mode.git ~/.atom/packages/activate-power-mode
```

Comment:
1) fork first, and then clone from contributor's codebase. here is the github official guide: https://guides.github.com/activities/contributing-to-open-source
2) according to https://atom.io/docs/v1.2.4/hacking-atom-package-word-count, cloning the code to the developer's workspace and link to ~/.atom/packages or ~/.atom/dev/packages is a better choice. On thing to note is if link the source code to ~/.atom/dev/packages, we need enter dev mode with `$ atom -d` to enable the plugin.